client/src/components/Profile/Profile.js

```diff
@@ -1,8 +1,39 @@
-import React from "react";
+import { React, useState } from "react";
 import PropTypes from "prop-types";
+import * as fetch from "../../lib/fetch";
 
 function Profile({ id }) {
-  return <>Profile of user with id {id}</>;
+  const [userExists, setUserExists] = useState(true);
+  const [name, setName] = useState("");
+  const [userID, setUserID] = useState("");
```

Comment:
What's the difference between `userID` and `id`?

Reply:
There's no difference logically speaking. Do you think we should remove `userID`?

Reply:
That's what I was thinking, yes